### ISSUE TITLE
Usertimeline v1 endpoint .done property value is false even after it has fetched all tweets

### ISSUE THREAD

**Issue by rishabhgarg7** [PLhery/node-twitter-api-v2#238]
Bug:

v1.userTimeline method's result done property has value equal to false even when it has fetched all tweets. I think it's only happening when there is only one page(haven't tested for multiple pages example yet)

To Reproduce:
1. Using read access client. 
2. Code:
```
const userTimeline = await client.v1.userTimeline(userID, {
    tweet_mode: "compat",
    count: 200,
    include_entities: false,
    exclude_replies: true,
    trim_user: true,
  });
userTimeline.done -> is giving false value
```
4.  Tested for this account: https://twitter.com/Rishabh12646510 (only have two tweets)

Expected behavior:
userTimeline.done -> should give true value

**Version**
 - Node.js version: 14.17.1
 - Lib version: 1.11.2
 - OS: MacOS

**Comment by alkihis** [comment]
Seems like a bug, but in v1 it could be quite hard to store the information that fetch is complete, without needing an dedicated boolean. I will take a look of how it could avoid it to happen.

**Comment by alkihis** [comment]
Should be fixed in 1.11.3.
